404.html

```diff
@@ -179,12 +179,22 @@ <h1>TAKE A MULLIGAN</h1>
     </main>
     <footer></footer>
     <script type="module">
-      import { createOptimizedPicture } from "./scripts/lib-franklin.js";
-      const imageUrl = "/media_10b8f0c21a329ee5ab764abf34d64a94de757ea2a.png";
-      const imageAlt = "Tiger Woods regretting a bad shot";
-      const pictureElement = createOptimizedPicture(imageUrl, imageAlt);
-      const imageContainer = document.querySelector(".error-image");
-      imageContainer.appendChild(pictureElement);
+      import { createOptimizedPicture } from "/scripts/lib-franklin.js";
+      import sitemap from "/import/tools/importer/data/sitemap.json" assert {type: 'json'}
+
+      const pathLocation = sitemap.find((obj) => Object.hasOwn(obj, window.location.pathname))
+      if (pathLocation) {
+        const drivePath = '/content-v2/' + pathLocation[window.location.pathname].replace('/content/golfdigest-com/en/', '').replace('.html', '')
+        window.location.assign(drivePath)
```

Comment:
If the user clicks on the back button, does he end up in an infinite loop in this case ?

Reply:
no, I have not found that to be the case